src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java

```diff
@@ -2513,9 +2513,36 @@ public void run3() {
+        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_Utility.getIntegratedCircuit(22), GT_OreDictUnificator.get(OrePrefixes.plate, Materials.Gold, 4L), GT_ModHandler.getModItem("IC2", "reactorVentDiamond", 1L, 1)}, Materials.StainlessSteel.getMolten(144), GT_ModHandler.getModItem("IC2", "reactorVentGold", 1L, 1),100, 480);
+        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_Utility.getIntegratedCircuit(21), GT_OreDictUnificator.get(OrePrefixes.plateDense, Materials.Tin, 4L), new ItemStack(Blocks.iron_bars, 4), GT_ModHandler.getModItem("IC2", "reactorVent", 1L, 1)}, GT_Values.NF, GT_ModHandler.getModItem("IC2", "reactorVentSpread", 1L), 100, 30);
+        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_Utility.getIntegratedCircuit(21), GT_OreDictUnificator.get(OrePrefixes.plate, Materials.RedAlloy, 6L), GT_ModHandler.getModItem("IC2", "reactorHeatSwitchCore", 2L, 1), GT_ModHandler.getModItem("IC2", "reactorVentCore", 1L, 1)}, GT_Values.NF, GT_ModHandler.getModItem("IC2", "reactorCondensator", 1L, 1), 100, 120);
+        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_Utility.getIntegratedCircuit(21), GT_OreDictUnificator.get(OrePrefixes.plateDense, Materials.Lapis, 1L), GT_OreDictUnificator.get(OrePrefixes.plateDense, Materials.RedAlloy, 4L), GT_ModHandler.getModItem("IC2", "reactorVentGold", 1L, 1), GT_ModHandler.getModItem("IC2", "reactorHeatSwitchSpread", 1L, 1), GT_ModHandler.getModItem("IC2", "reactorCondensator", 2L, 1)}, GT_Values.NF, GT_ModHandler.getModItem("IC2", "reactorCondensatorLap", 1L, 1), 100, 120);
+
+        //Reactor Plating
+        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_Utility.getIntegratedCircuit(22), GT_OreDictUnificator.get(OrePrefixes.plateDense, Materials.Lead, 1L), GT_ModHandler.getModItem("IC2", "itemPartAlloy", 8L)}, GT_Values.NF, GT_ModHandler.getModItem("IC2", "reactorPlating", 1L), 100, 120);
+        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_Utility.getIntegratedCircuit(21), GT_OreDictUnificator.get(OrePrefixes.plateDense, Materials.Copper, 4L), GT_OreDictUnificator.get(OrePrefixes.plate, Materials.Copper, 2L), GT_OreDictUnificator.get(OrePrefixes.plate, Materials.Silver, 2L), GT_ModHandler.getModItem("IC2", "reactorPlating", 1L)}, GT_Values.NF, GT_ModHandler.getModItem("IC2", "reactorPlatingHeat", 1L), 100, 120);
+        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_Utility.getIntegratedCircuit(23), GT_OreDictUnificator.get(OrePrefixes.plateDense, Materials.Lead, 4L), GT_ModHandler.getModItem("IC2", "itemPartAlloy", 4L), GT_ModHandler.getModItem("IC2", "reactorPlating", 1L)}, GT_Values.NF, GT_ModHandler.getModItem("IC2", "reactorPlatingExplosive", 1L), 100, 120);
+
+        //LV and MV Energy Hatches
```

Comment:
if you put lv and mv energy to code you should add the rest as well.

Reply:
The HV hatch and higher already have assembler recipes and can't be crafted without them. The MV and LV don't have those recipes, so I added them.